src/analysis_file.py

```diff
@@ -41,16 +76,17 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
 
                     if cc.folding_mode == FoldingModes.ASSERT_EQUAL:
                         fold_strategies[cc.coded_field] = FoldStrategies.assert_label_ids_equal
-                        fold_strategies[cc.analysis_file_key] = FoldStrategies.assert_equal
                     elif cc.folding_mode == FoldingModes.YES_NO_AMB:
-                        fold_strategies[cc.analysis_file_key] = FoldStrategies.yes_no_amb
```

Comment:
add a TODO here, please.

Reply:
Have added an issue: #78

(this isn't actually needed for OCHA, but will be when applied to IMAQAL)